container_workflow_tool/distgit.py

```diff
@@ -109,10 +117,11 @@ def _get_from(self, fdata):
         Returns:
             str: FROM string
         """
+        registry_base = None
         image_base = re.search('FROM (.*)\n', fdata)
         if image_base:
-            image_base = image_base.group(1)
-        return image_base
+            registry_base = image_base.group(1)
+        return registry_base
```

Comment:
I guess this is to make sure we always return `None` instead of whatever `re.search` returns when no entries are found? Good idea, thanks!

Reply:
Yes. It was catched, by testing.

Reply:
We would like to return either None or string.

Reply:
image_base is regexp string.